pokedex/data/csv/machines.csv

```diff
@@ -1441,3 +1608,96 @@ machine_number,version_group_id,item_id,move_id
 108,10,404,431
 108,12,404,291
 108,13,404,291
+108,20,1196,67
+109,20,1197,85
+110,20,1198,87
+111,20,1199,89
+112,20,1200,94
+113,20,1201,97
+114,20,1202,116
+115,20,1203,118
+116,20,1204,126
+117,20,1205,127
+118,20,1206,133
+119,20,1207,141
+120,20,1208,161
+121,20,1209,164
+122,20,1210,179
+123,20,1211,188
+124,20,1212,191
+125,20,1213,200
+126,20,1214,473
+127,20,1215,203
+128,20,1216,214
+129,20,1217,224
+130,20,1218,226
+131,20,1219,227
+132,20,1220,231
+133,20,1221,242
+134,20,1222,247
+135,20,1223,248
+136,20,1224,253
+137,20,1225,257
+138,20,1226,269
+139,20,1227,271
+140,20,1228,276
+141,20,1229,285
+142,20,1230,299
+143,20,1231,304
+144,20,1232,315
+145,20,1233,322
+146,20,1234,330
+147,20,1235,334
+148,20,1236,337
+149,20,1237,339
+150,20,1238,347
+151,20,1239,348
+152,20,1240,349
+153,20,1241,360
+154,20,1242,370
+155,20,1243,390
+156,20,1244,394
+157,20,1245,396
+158,20,1246,398
+159,20,1247,399
+160,20,1248,402
+161,20,1249,404
+162,20,1250,405
+163,20,1251,406
+164,20,1252,408
+165,20,1253,411
+166,20,1254,412
+167,20,1255,413
+168,20,1256,414
+169,20,1257,417
+170,20,1258,428
+171,20,1259,430
+172,20,1260,437
+173,20,1261,438
+174,20,1262,441
+175,20,1263,442
+176,20,1264,444
+177,20,1265,446
+178,20,1266,447
+179,20,1267,482
+180,20,1268,484
+181,20,1269,486
+182,20,1270,492
+183,20,1271,500
+184,20,1272,502
+185,20,1273,503
+186,20,1274,526
+187,20,1275,528
+188,20,1276,529
+189,20,1277,535
+190,20,1278,542
+191,20,1279,583
+192,20,1280,599
+193,20,1281,605
+194,20,1282,663
+195,20,1283,667
+196,20,1284,675
+197,20,1285,676
+198,20,1286,706
+199,20,1287,710
+200,20,1188,776
```

Comment:
TMs and TRs are numbered from 00 to 99, here i used IDs 1-100 for the TMs, and 101-200 for the TRs, so there's a difference of one between the two numberings. For example TM05 is ID 6, and TR99 is ID 200.

Reply:
Shifted them down, it doesn't seem to cause problems using sqlite, not sure about others